src/lambdas/auditMessageFirehoseReingest/getAuditEvents.test.ts

```diff
@@ -0,0 +1,67 @@
+import { when } from 'jest-when'
+import { getAuditEvents } from './getAuditEvents'
+import { getAuditEventsFromS3Object } from './getAuditEventsFromS3Object'
+
+jest.mock('./getAuditEventsFromS3Object', () => ({
+  getAuditEventsFromS3Object: jest.fn()
+}))
+
+describe('getAuditEvents', () => {
+  const s3ObjectDetails = [
+    {
+      bucket: 'mockBucket',
+      key: 'failures/mockKey1',
+      sqsRecordMessageId: 'mockMessageId1'
+    },
+    {
+      bucket: 'mockBucket',
+      key: 'failures/mockKey2',
+      sqsRecordMessageId: 'mockMessageId2'
+    }
+  ]
+  const auditEvents = [
+    {
+      event_name: 'MOCK_EVENT_NAME',
+      timestamp: 12345678
+    }
+  ]
+
+  it('returns the audit events from the provided s3 objects', async () => {
+    when(getAuditEventsFromS3Object).mockResolvedValue(auditEvents)
+
+    const expectedResult = {
+      successfulResults: [
+        {
+          auditEvents,
+          bucket: 'mockBucket',
+          key: 'failures/mockKey1',
```

Comment:
If you're setting up an assertion like this, I'd rather we didn't just copy and paste strings, and just set up some constants somewhere (probably just in the same file)

Reply:
Done